// @flesh-and-blood/types/dist/interfaces.js
var Foiling = /* @__PURE__ */ ((Foiling2) => {
  Foiling2["Cold"] = "Cold";
  Foiling2["Gold"] = "Gold";
  Foiling2["Rainbow"] = "Rainbow";
  return Foiling2;
})(Foiling || {});
var Shorthand = /* @__PURE__ */ ((Shorthand2) => {
  Shorthand2["Buffs"] = "Buffs";
  Shorthand2["Flick"] = "Flick";
  Shorthand2["Poppers"] = "Poppers";
  Shorthand2["Tap"] = "Tap";
  Shorthand2["Untap"] = "Untap";
  return Shorthand2;
})(Shorthand || {});
var Treatment = /* @__PURE__ */ ((Treatment2) => {
  Treatment2["AB"] = "Alternate Border";
  Treatment2["AA"] = "Alternate Art";
  Treatment2["AT"] = "Alternate Text";
  Treatment2["EA"] = "Extended Art";
  Treatment2["FA"] = "Full Art";
  return Treatment2;
})(Treatment || {});
var Class = /* @__PURE__ */ ((Class2) => {
  Class2["NotClassed"] = "NotClassed";
  Class2["Generic"] = "Generic";
  Class2["Adjudicator"] = "Adjudicator";
  Class2["Assassin"] = "Assassin";
  Class2["Bard"] = "Bard";
  Class2["Brute"] = "Brute";
  Class2["Guardian"] = "Guardian";
  Class2["Illusionist"] = "Illusionist";
  Class2["Mechanologist"] = "Mechanologist";
  Class2["Merchant"] = "Merchant";
  Class2["Necromancer"] = "Necromancer";
  Class2["Ninja"] = "Ninja";
  Class2["Pirate"] = "Pirate";
  Class2["Ranger"] = "Ranger";
  Class2["Runeblade"] = "Runeblade";
  Class2["Shapeshifter"] = "Shapeshifter";
  Class2["Thief"] = "Thief";
  Class2["Warrior"] = "Warrior";
  Class2["Wizard"] = "Wizard";
  return Class2;
})(Class || {});
var Bond = /* @__PURE__ */ ((Bond2) => {
  Bond2["Earth"] = "Earth";
  return Bond2;
})(Bond || {});
var Flow = /* @__PURE__ */ ((Flow2) => {
  Flow2["Lightning"] = "Lightning";
  return Flow2;
})(Flow || {});
var Format = /* @__PURE__ */ ((Format2) => {
  Format2["Blitz"] = "Blitz";
  Format2["ClassicConstructed"] = "Classic Constructed";
  Format2["Draft"] = "Draft";
  Format2["GoldenAge"] = "Golden Age";
  Format2["LivingLegend"] = "Living Legend";
  Format2["Open"] = "Open";
  Format2["Sealed"] = "Sealed";
  Format2["SilverAge"] = "Silver Age";
  Format2["UltimatePitFight"] = "Ultimate Pit Fight";
  return Format2;
})(Format || {});
var Fusion = /* @__PURE__ */ ((Fusion2) => {
  Fusion2["Earth"] = "Earth";
  Fusion2["Ice"] = "Ice";
  Fusion2["Lightning"] = "Lightning";
  return Fusion2;
})(Fusion || {});
var Rarity = /* @__PURE__ */ ((Rarity2) => {
  Rarity2["Basic"] = "Basic";
  Rarity2["Token"] = "Token";
  Rarity2["Common"] = "Common";
  Rarity2["Rare"] = "Rare";
  Rarity2["SuperRare"] = "Super Rare";
  Rarity2["Majestic"] = "Majestic";
  Rarity2["Marvel"] = "Marvel";
  Rarity2["Legendary"] = "Legendary";
  Rarity2["Fabled"] = "Fabled";
  Rarity2["Promo"] = "Promo";
  return Rarity2;
})(Rarity || {});
var Meta = /* @__PURE__ */ ((Meta2) => {
  Meta2["Expansion"] = "Expansion slot";
  Meta2["Rainbow"] = "Rainbow";
  return Meta2;
})(Meta || {});
var Release = /* @__PURE__ */ ((Release2) => {
  Release2["ArcaneRising"] = "Arcane Rising";
  Release2["BrightLights"] = "Bright Lights";
  Release2["CompendiumOfRathe"] = "Compendium of Rathe";
  Release2["CrucibleOfWar"] = "Crucible of War";
  Release2["DuskTillDawn"] = "Dusk till Dawn";
  Release2["Dynasty"] = "Dynasty";
  Release2["Everfest"] = "Everfest";
  Release2["HeavyHitters"] = "Heavy Hitters";
  Release2["HighSeas"] = "High Seas";
  Release2["HistoryPack1"] = "History Pack 1";
  Release2["IAR"] = "IAR";
  Release2["Monarch"] = "Monarch";
  Release2["OmensOfTheThirdAge"] = "Omens of the Third Age";
  Release2["Outsiders"] = "Outsiders";
  Release2["PartTheMistveil"] = "Part the Mistveil";
  Release2["Rosetta"] = "Rosetta";
  Release2["SuperSlam"] = "Super Slam";
  Release2["TalesOfAria"] = "Tales of Aria";
  Release2["TheHunted"] = "The Hunted";
  Release2["Uprising"] = "Uprising";
  Release2["WelcomeToRathe"] = "Welcome to Rathe";
  Release2["ArmoryDeckArakni"] = "Armory Deck: Arakni";
  Release2["ArmoryDeckAurora"] = "Armory Deck: Aurora";
  Release2["ArmoryDeckAzalea"] = "Armory Deck: Azalea";
  Release2["ArmoryDeckBoltyn"] = "Armory Deck: Boltyn";
  Release2["ArmoryDeckDash"] = "Armory Deck: Dash";
  Release2["ArmoryDeckGravyBones"] = "Armory Deck: Gravy Bones";
  Release2["ArmoryDeckIra"] = "Armory Deck: Ira";
  Release2["ArmoryDeckKayo"] = "Armory Deck: Kayo";
  Release2["ArmoryDeckMaxx"] = "Armory Deck: Maxx";
  Release2["ArmoryDeckOlympia"] = "Armory Deck: Olympia";
  Release2["ArmoryDeckPleiades"] = "Armory Deck: Pleiades";
  Release2["ArmoryDeckRhinar"] = "Armory Deck: Rhinar";
  Release2["ArmoryDeckZyggy"] = "Armory Deck: Zyggy";
  Release2["ArmoryDeckOriginsHala"] = "Armory Deck Origins: Hala";
  Release2["ArmoryDeckOriginsJarl"] = "Armory Deck Origins: Jarl";
  Release2["ArmoryDeckLegendsPrism"] = "Armory Deck Legends: Prism";
  Release2["ArmoryDeckLegendsViserai"] = "Armory Deck Legends: Viserai";
  Release2["ArakniBlitzDeck"] = "Arakni Blitz Deck";
  Release2["ArakniBlitzDeckARK"] = "Arakni Hunted Blitz Deck";
  Release2["ArakniWebOfDeceitBlitzDeck"] = "Arakni, Web of Deceit Blitz Deck";
  Release2["AuroraBlitzDeck"] = "Aurora Blitz Deck";
  Release2["AzaleaBlitzDeck"] = "Azalea Blitz Deck";
  Release2["BenjiBlitzDeck"] = "Benji Blitz Deck";
  Release2["BetsyBlitzDeck"] = "Betsy Blitz Deck";
  Release2["BoltynBlitzDeck"] = "Boltyn Blitz Deck";
  Release2["BriarBlitzDeck"] = "Briar Blitz Deck";
  Release2["ChaneBlitzDeck"] = "Chane Blitz Deck";
  Release2["CindraBlitzDeck"] = "Cindra Blitz Deck";
  Release2["DromaiBlitzDeck"] = "Dromai Blitz Deck";
  Release2["EnigmaBlitzDeck"] = "Enigma Blitz Deck";
  Release2["FaiBlitzDeck"] = "Fai Blitz Deck";
  Release2["FangBlitzDeck"] = "Fang Blitz Deck";
  Release2["FlorianBlitzDeck"] = "Florian Blitz Deck";
  Release2["KassaiBlitzDeck"] = "Kassai Blitz Deck";
  Release2["KatsuBlitzDeck"] = "Katsu Blitz Deck";
  Release2["KayoBlitzDeck"] = "Kayo Blitz Deck";
  Release2["LeviaBlitzDeck"] = "Levia Blitz Deck";
  Release2["LexiBlitzDeck"] = "Lexi Blitz Deck";
  Release2["NuuBlitzDeck"] = "Nuu Blitz Deck";
  Release2["OldhimBlitzDeck"] = "Oldhim Blitz Deck";
  Release2["OlympiaBlitzDeck"] = "Olympia Blitz Deck";
  Release2["OscilioBlitzDeck"] = "Oscilio Blitz Deck";
  Release2["PrismBlitzDeck"] = "Prism Blitz Deck";
  Release2["RhinarBlitzDeck"] = "Rhinar Blitz Deck";
  Release2["RiptideBlitzDeck"] = "Riptide Blitz Deck";
  Release2["UzuriBlitzDeck"] = "Uzuri Blitz Deck";
  Release2["VerdanceBlitzDeck"] = "Verdance Blitz Deck";
  Release2["VictorBlitzDeck"] = "Victor Blitz Deck";
  Release2["ZenBlitzDeck"] = "Zen Blitz Deck";
  Release2["FirstStrikeAurora"] = "1st Strike: Aurora";
  Release2["FirstStrikeTerra"] = "1st Strike: Terra";
  Release2["BravoHeroDeck"] = "Bravo Hero Deck";
  Release2["DorintheaHeroDeck"] = "Dorinthea Hero Deck";
  Release2["KatsuHeroDeck"] = "Katsu Hero Deck";
  Release2["RhinarHeroDeck"] = "Rhinar Hero Deck";
  Release2["ArakniSilverAgeDeck"] = "Arakni Silver Age Deck";
  Release2["AzaleaSilverAgeDeck"] = "Azalea Silver Age Deck";
  Release2["BlazeSilverAgeDeck"] = "Blaze Silver Age Deck";
  Release2["BoltynSilverAgeDeck"] = "Boltyn Silver Age Deck";
  Release2["BravoSilverAgeDeck"] = "Bravo Silver Age Deck";
  Release2["BriarSilverAgeDeck"] = "Briar Silver Age Deck";
  Release2["DashSilverAgeDeck"] = "Dash Silver Age Deck";
  Release2["DorintheaSilverAgeDeck"] = "Dorinthea Silver Age Deck";
  Release2["EnigmaSilverAgeDeck"] = "Enigma Silver Age Deck";
  Release2["FaiSilverAgeDeck"] = "Fai Silver Age Deck";
  Release2["GravyBonesSilverAgeDeck"] = "Gravy Bones Silver Age Deck";
  Release2["IyslanderSilverAgeDeck"] = "Iyslander Silver Age Deck";
  Release2["KayoSilverAgeDeck"] = "Kayo Silver Age Deck";
  Release2["LyathSilverAgeDeck"] = "Lyath Silver Age Deck";
  Release2["ViseraiSilverAgeDeck"] = "Viserai Silver Age Deck";
  Release2["MasteryPackGuardian"] = "Mastery Pack: Guardian";
  Release2["MasteryPackWarrior"] = "Mastery Pack: Warrior";
  Release2["BravoDemoDeck"] = "Bravo Demo Deck";
  Release2["DorintheaDemoDeck"] = "Dorinthea Demo Deck";
  Release2["ClassicBattlesRhinarDorinthea"] = "Classic Battles: Rhinar vs Dorinthea";
  Release2["IraWelcomeDeck"] = "Ira Welcome Deck";
  Release2["RoundTheTable"] = "Round the Table: TCC x LSS";
  Release2["SmashPalace"] = "Smash Palace";
  Release2["GEM"] = "GEM";
  Release2["Promos"] = "Promos";
  Release2["TournamentPack"] = "Tournament Pack";
  return Release2;
})(Release || {});
var ReleaseEdition = /* @__PURE__ */ ((ReleaseEdition2) => {
  ReleaseEdition2["Alpha"] = "Alpha";
  ReleaseEdition2["First"] = "First";
  ReleaseEdition2["Unlimited"] = "Unlimited";
  ReleaseEdition2["Promo"] = "Promo";
  return ReleaseEdition2;
})(ReleaseEdition || {});
var Talent = /* @__PURE__ */ ((Talent2) => {
  Talent2["Chaos"] = "Chaos";
  Talent2["Draconic"] = "Draconic";
  Talent2["Earth"] = "Earth";
  Talent2["Elemental"] = "Elemental";
  Talent2["Ice"] = "Ice";
  Talent2["Light"] = "Light";
  Talent2["Lightning"] = "Lightning";
  Talent2["Mystic"] = "Mystic";
  Talent2["Revered"] = "Revered";
  Talent2["Reviled"] = "Reviled";
  Talent2["Royal"] = "Royal";
  Talent2["Shadow"] = "Shadow";
  return Talent2;
})(Talent || {});
var Trait = /* @__PURE__ */ ((Trait2) => {
  Trait2["AgentOfChaos"] = "Agent of Chaos";
  return Trait2;
})(Trait || {});
var Metatype = /* @__PURE__ */ ((Metatype2) => {
  Metatype2["Arakni"] = "Arakni";
  Metatype2["OmensOfTheThirdAge"] = "Omens of the Third Age";
  Metatype2["Scurv"] = "Scurv";
  Metatype2["HighSeas"] = "High Seas";
  Metatype2["Puffin"] = "Puffin";
  Metatype2["Rosetta"] = "Rosetta";
  return Metatype2;
})(Metatype || {});
var Type = /* @__PURE__ */ ((Type2) => {
  Type2["Action"] = "Action";
  Type2["AttackReaction"] = "Attack Reaction";
  Type2["Block"] = "Block";
  Type2["Companion"] = "Companion";
  Type2["DefenseReaction"] = "Defense Reaction";
  Type2["DemiHero"] = "Demi-Hero";
  Type2["Equipment"] = "Equipment";
  Type2["Hero"] = "Hero";
  Type2["Instant"] = "Instant";
  Type2["Macro"] = "Macro";
  Type2["Mentor"] = "Mentor";
  Type2["Resource"] = "Resource";
  Type2["Token"] = "Token";
  Type2["Weapon"] = "Weapon";
  return Type2;
})(Type || {});
var Subtype = /* @__PURE__ */ ((Subtype2) => {
  Subtype2["OneHanded"] = "1H";
  Subtype2["TwoHanded"] = "2H";
  Subtype2["Affliction"] = "Affliction";
  Subtype2["Ally"] = "Ally";
  Subtype2["Angel"] = "Angel";
  Subtype2["Arms"] = "Arms";
  Subtype2["Arrow"] = "Arrow";
  Subtype2["Ash"] = "Ash";
  Subtype2["Attack"] = "Attack";
  Subtype2["Aura"] = "Aura";
  Subtype2["Axe"] = "Axe";
  Subtype2["Base"] = "Base";
  Subtype2["Book"] = "Book";
  Subtype2["Bow"] = "Bow";
  Subtype2["Cannon"] = "Cannon";
  Subtype2["Chest"] = "Chest";
  Subtype2["Chi"] = "Chi";
  Subtype2["Claw"] = "Claw";
  Subtype2["Club"] = "Club";
  Subtype2["Construct"] = "Construct";
  Subtype2["Dagger"] = "Dagger";
  Subtype2["Demon"] = "Demon";
  Subtype2["Dragon"] = "Dragon";
  Subtype2["Evo"] = "Evo";
  Subtype2["Fiddle"] = "Fiddle";
  Subtype2["Figment"] = "Figment";
  Subtype2["Flail"] = "Flail";
  Subtype2["Gem"] = "Gem";
  Subtype2["Gun"] = "Gun";
  Subtype2["Hammer"] = "Hammer";
  Subtype2["Head"] = "Head";
  Subtype2["Invocation"] = "Invocation";
  Subtype2["Item"] = "Item";
  Subtype2["Landmark"] = "Landmark";
  Subtype2["Log"] = "Log";
  Subtype2["Lute"] = "Lute";
  Subtype2["Legs"] = "Legs";
  Subtype2["NonAttack"] = "Non-Attack";
  Subtype2["OffHand"] = "Off-Hand";
  Subtype2["Orb"] = "Orb";
  Subtype2["Pistol"] = "Pistol";
  Subtype2["PitFighter"] = "Pit-Fighter";
  Subtype2["Polearm"] = "Polearm";
  Subtype2["Quiver"] = "Quiver";
  Subtype2["Rock"] = "Rock";
  Subtype2["Shuriken"] = "Shuriken";
  Subtype2["Scepter"] = "Scepter";
  Subtype2["Scroll"] = "Scroll";
  Subtype2["Scythe"] = "Scythe";
  Subtype2["Song"] = "Song";
  Subtype2["Staff"] = "Staff";
  Subtype2["Sword"] = "Sword";
  Subtype2["Trap"] = "Trap";
  Subtype2["Wrench"] = "Wrench";
  Subtype2["Young"] = "Young";
  return Subtype2;
})(Subtype || {});
var Hero = /* @__PURE__ */ ((Hero2) => {
  Hero2["Arakni"] = "Arakni";
  Hero2["Aurora"] = "Aurora";
  Hero2["Aurora2"] = "Aurora2";
  Hero2["Azalea"] = "Azalea";
  Hero2["Baalghor"] = "Baalghor";
  Hero2["Benji"] = "Benji";
  Hero2["Betsy"] = "Betsy";
  Hero2["Blaze"] = "Blaze";
  Hero2["Bolfar"] = "Bolfar";
  Hero2["Boltyn"] = "Boltyn";
  Hero2["Bravo"] = "Bravo";
  Hero2["Brevant"] = "Brevant";
  Hero2["Briar"] = "Briar";
  Hero2["Brutus"] = "Brutus";
  Hero2["Chane"] = "Chane";
  Hero2["Cindra"] = "Cindra";
  Hero2["Crackni"] = "Crackni";
  Hero2["Crix"] = "Crix";
  Hero2["Dash"] = "Dash";
  Hero2["DataDoll"] = "Data Doll";
  Hero2["Dorinthea"] = "Dorinthea";
  Hero2["Dromai"] = "Dromai";
  Hero2["Emperor"] = "Emperor";
  Hero2["Enigma"] = "Enigma";
  Hero2["Fai"] = "Fai";
  Hero2["Fang"] = "Fang";
  Hero2["Florian"] = "Florian";
  Hero2["Frankie"] = "Frankie";
  Hero2["Genis"] = "Genis";
  Hero2["GravyBones"] = "Gravy Bones";
  Hero2["Hala"] = "Hala";
  Hero2["Ira"] = "Ira";
  Hero2["Iyslander"] = "Iyslander";
  Hero2["Jarl"] = "Jarl";
  Hero2["Kano"] = "Kano";
  Hero2["Kassai"] = "Kassai";
  Hero2["Katsu"] = "Katsu";
  Hero2["Kavdaen"] = "Kavdaen";
  Hero2["Kayo"] = "Kayo";
  Hero2["Kox"] = "Kox";
  Hero2["Levia"] = "Levia";
  Hero2["Lexi"] = "Lexi";
  Hero2["Librarian"] = "Librarian";
  Hero2["Lyath"] = "Lyath";
  Hero2["Marlynn"] = "Marlynn";
  Hero2["Maxx"] = "Maxx";
  Hero2["Melody"] = "Melody";
  Hero2["Nuu"] = "Nuu";
  Hero2["Oldhim"] = "Oldhim";
  Hero2["Olympia"] = "Olympia";
  Hero2["Oscilio"] = "Oscilio";
  Hero2["Broscilio"] = "Broscilio";
  Hero2["Pleiades"] = "Pleiades";
  Hero2["Prism"] = "Prism";
  Hero2["Puffin"] = "Puffin";
  Hero2["Reya"] = "Reya";
  Hero2["Rhinar"] = "Rhinar";
  Hero2["Riptide"] = "Riptide";
  Hero2["RKO"] = "RKO";
  Hero2["Ruudi"] = "Ruu'di";
  Hero2["Shiyana"] = "Shiyana";
  Hero2["Slippy"] = "Slippy";
  Hero2["Squizzy"] = "Squizzy";
  Hero2["Scurv"] = "Scurv";
  Hero2["Starvo"] = "Starvo";
  Hero2["Taipanis"] = "Taipanis";
  Hero2["Taylor"] = "Taylor";
  Hero2["Teklovossen"] = "Teklovossen";
  Hero2["Terra"] = "Terra";
  Hero2["Theryon"] = "Theryon";
  Hero2["Tuffnut"] = "Tuffnut";
  Hero2["Uzuri"] = "Uzuri";
  Hero2["Valda"] = "Valda";
  Hero2["Verdance"] = "Verdance";
  Hero2["Victor"] = "Victor";
  Hero2["Vynnset"] = "Vynnset";
  Hero2["Viserai"] = "Viserai";
  Hero2["Yorick"] = "Yorick";
  Hero2["Yoji"] = "Yoji";
  Hero2["Zen"] = "Zen";
  Hero2["Zyggy"] = "Zyggy";
  return Hero2;
})(Hero || {});
var Keyword = /* @__PURE__ */ ((Keyword2) => {
  Keyword2["Ambush"] = "Ambush";
  Keyword2["Amp"] = "Amp";
  Keyword2["ArcaneBarrier"] = "Arcane Barrier";
  Keyword2["ArcaneShelter"] = "Arcane Shelter";
  Keyword2["Awaken"] = "Awaken";
  Keyword2["Battleworn"] = "Battleworn";
  Keyword2["BeatChest"] = "Beat Chest";
  Keyword2["BladeBreak"] = "Blade Break";
  Keyword2["BloodDebt"] = "Blood Debt";
  Keyword2["Boost"] = "Boost";
  Keyword2["Bond"] = "Bond";
  Keyword2["Channel"] = "Channel";
  Keyword2["Charge"] = "Charge";
  Keyword2["Clash"] = "Clash";
  Keyword2["Cloaked"] = "Cloaked";
  Keyword2["Combo"] = "Combo";
  Keyword2["Contract"] = "Contract";
  Keyword2["Crank"] = "Crank";
  Keyword2["TheCrowdBoos"] = "The Crowd Boos";
  Keyword2["TheCrowdCheers"] = "The Crowd Cheers";
  Keyword2["Crush"] = "Crush";
  Keyword2["Decompose"] = "Decompose";
  Keyword2["Dominate"] = "Dominate";
  Keyword2["Ephemeral"] = "Ephemeral";
  Keyword2["Essence"] = "Essence";
  Keyword2["EvoUpgrade"] = "Evo Upgrade";
  Keyword2["Flow"] = "Flow";
  Keyword2["Fragment"] = "Fragment";
  Keyword2["Freeze"] = "Freeze";
  Keyword2["Fusion"] = "Fusion";
  Keyword2["Galvanize"] = "Galvanize";
  Keyword2["GoAgain"] = "Go again";
  Keyword2["GoFish"] = "Go Fish";
  Keyword2["Guardwell"] = "Guardwell";
  Keyword2["Heave"] = "Heave";
  Keyword2["Heavy"] = "Heavy";
  Keyword2["HighTide"] = "High Tide";
  Keyword2["Intimidate"] = "Intimidate";
  Keyword2["Legendary"] = "Legendary";
  Keyword2["Mark"] = "Mark";
  Keyword2["Material"] = "Material";
  Keyword2["Meld"] = "Meld";
  Keyword2["Modular"] = "Modular";
  Keyword2["Mirage"] = "Mirage";
  Keyword2["Negate"] = "Negate";
  Keyword2["Opt"] = "Opt";
  Keyword2["Overpower"] = "Overpower";
  Keyword2["Pairs"] = "Pairs";
  Keyword2["Piercing"] = "Piercing";
  Keyword2["Phantasm"] = "Phantasm";
  Keyword2["Protect"] = "Protect";
  Keyword2["Quell"] = "Quell";
  Keyword2["Quickstrike"] = "Quickstrike";
  Keyword2["Reload"] = "Reload";
  Keyword2["Reprise"] = "Reprise";
  Keyword2["Retrieve"] = "Retrieve";
  Keyword2["RuneGate"] = "Rune Gate";
  Keyword2["Rupture"] = "Rupture";
  Keyword2["Scrap"] = "Scrap";
  Keyword2["Sharpen"] = "Sharpen";
  Keyword2["Solflare"] = "Solflare";
  Keyword2["Specialization"] = "Specialization";
  Keyword2["Spectra"] = "Spectra";
  Keyword2["Spellvoid"] = "Spellvoid";
  Keyword2["Starfall"] = "Starfall";
  Keyword2["Steal"] = "Steal";
  Keyword2["Stealth"] = "Stealth";
  Keyword2["Surge"] = "Surge";
  Keyword2["Suspense"] = "Suspense";
  Keyword2["Temper"] = "Temper";
  Keyword2["Tower"] = "Tower";
  Keyword2["Transform"] = "Transform";
  Keyword2["Transcend"] = "Transcend";
  Keyword2["Unlimited"] = "Unlimited";
  Keyword2["Universal"] = "Universal";
  Keyword2["Unfreeze"] = "Unfreeze";
  Keyword2["Unity"] = "Unity";
  Keyword2["Usurp"] = "Usurp";
  Keyword2["Wager"] = "Wager";
  Keyword2["Ward"] = "Ward";
  Keyword2["WateryGrave"] = "Watery Grave";
  return Keyword2;
})(Keyword || {});
export {
  Bond,
  Class,
  Flow,
  Foiling,
  Format,
  Fusion,
  Hero,
  Keyword,
  Meta,
  Metatype,
  Rarity,
  Release,
  ReleaseEdition,
  Shorthand,
  Subtype,
  Talent,
  Trait,
  Treatment,
  Type
};
