// @flesh-and-blood/types/dist/sets.js
import { Class, Hero, Rarity, Release, Talent } from "./interfaces.js";
var Language = /* @__PURE__ */ ((Language2) => {
  Language2["English"] = "English";
  Language2["French"] = "Fran\xE7ais";
  Language2["German"] = "Deutsch";
  Language2["Italian"] = "Italiano";
  Language2["Japanese"] = "\u65E5\u672C\u8A9E";
  Language2["Spanish"] = "Espa\xF1ol";
  return Language2;
})(Language || {});
const ALL_LANGUAGES_OLD = [
  "English" /* English */,
  "Fran\xE7ais" /* French */,
  "Deutsch" /* German */,
  "Italiano" /* Italian */,
  "\u65E5\u672C\u8A9E" /* Japanese */,
  "Espa\xF1ol" /* Spanish */
];
const ALL_LANGUAGES = ["English" /* English */, "Fran\xE7ais" /* French */, "\u65E5\u672C\u8A9E" /* Japanese */];
var ReleaseType = /* @__PURE__ */ ((ReleaseType2) => {
  ReleaseType2["ArmoryDeck"] = "Armory Deck";
  ReleaseType2["BlitzDeck"] = "Blitz Deck";
  ReleaseType2["BoxSet"] = "Box Set";
  ReleaseType2["ClassicBattles"] = "Classic Battles";
  ReleaseType2["ExpansionBooster"] = "Expansion Booster";
  ReleaseType2["FirstStrike"] = "1st Strike";
  ReleaseType2["HistoryPack"] = "History Pack";
  ReleaseType2["MasteryPack"] = "Mastery Pack";
  ReleaseType2["RoundTheTable"] = "Round the Table";
  ReleaseType2["SilverAgeDeck"] = "Silver Age Deck";
  ReleaseType2["StandaloneBooster"] = "Standalone Booster";
  ReleaseType2["WelcomeDeck"] = "Welcome Deck";
  return ReleaseType2;
})(ReleaseType || {});
const BOOSTER_RELEASE_TYPES = [
  "Expansion Booster" /* ExpansionBooster */,
  "Standalone Booster" /* StandaloneBooster */
];
const getIsBooster = (releaseType) => BOOSTER_RELEASE_TYPES.includes(releaseType);
const getIsReprint = (releaseType) => releaseType === "History Pack" /* HistoryPack */;
const getIsDraftable = (releaseType) => releaseType === "Standalone Booster" /* StandaloneBooster */;
const PRECONSTRUCTED_RELEASE_TYPES = [
  "Armory Deck" /* ArmoryDeck */,
  "Blitz Deck" /* BlitzDeck */,
  "Classic Battles" /* ClassicBattles */,
  "1st Strike" /* FirstStrike */,
  "Round the Table" /* RoundTheTable */,
  "Silver Age Deck" /* SilverAgeDeck */,
  "Welcome Deck" /* WelcomeDeck */
];
const getIsPreconstructed = (releaseType) => PRECONSTRUCTED_RELEASE_TYPES.includes(releaseType);
const releases = [
  {
    cards: 30,
    classes: [Class.Ninja],
    deckLinks: [
      { url: "https://fabrary.net/decks/01GJG7Z4WGWSZ95FY74KX4M557" }
    ],
    heroes: [Hero.Ira],
    languages: ALL_LANGUAGES_OLD,
    release: Release.IraWelcomeDeck,
    relatedReleases: [],
    releaseDate: "2019-08-31 12:00",
    releaseType: "Welcome Deck" /* WelcomeDeck */,
    setIdentifiers: ["ira"],
    talents: []
  },
  {
    cards: 226,
    classes: [Class.Brute, Class.Guardian, Class.Ninja, Class.Warrior],
    deckLinks: [],
    draft: {
      heroIdentifiers: ["bravo", "dorinthea", "katsu", "rhinar"],
      picksPerPack: 15
    },
    heroes: [Hero.Bravo, Hero.Dorinthea, Hero.Katsu, Hero.Rhinar],
    languages: ["English" /* English */],
    release: Release.WelcomeToRathe,
    relatedReleases: [],
    releaseDate: "2019-10-11 12:00",
    releaseType: "Standalone Booster" /* StandaloneBooster */,
    setIdentifiers: ["wtr"],
    talents: []
  },
  {
    cards: 219,
    classes: [Class.Mechanologist, Class.Ranger, Class.Runeblade, Class.Wizard],
    deckLinks: [],
    draft: {
      heroIdentifiers: ["azalea", "dash", "kano", "viserai"],
      picksPerPack: 15
    },
    heroes: [Hero.Azalea, Hero.Dash, Hero.Kano, Hero.Viserai],
    languages: ["English" /* English */],
    release: Release.ArcaneRising,
    relatedReleases: [],
    releaseDate: "2020-03-27 12:00",
    releaseType: "Standalone Booster" /* StandaloneBooster */,
    setIdentifiers: ["arc"],
    talents: []
  },
  {
    cards: 194,
    classes: [
      Class.Brute,
      Class.Guardian,
      Class.Mechanologist,
      Class.Merchant,
      Class.Ninja,
      Class.Ranger,
      Class.Runeblade,
      Class.Warrior,
      Class.Wizard
    ],
    deckLinks: [],
    heroes: [
      Hero.Benji,
      Hero.DataDoll,
      Hero.Ira,
      Hero.Kassai,
      Hero.Kavdaen,
      Hero.Kayo,
      Hero.Shiyana
    ],
    languages: ["English" /* English */],
    release: Release.CrucibleOfWar,
    relatedReleases: [],
    releaseDate: "2020-08-28 12:00",
    releaseType: "Expansion Booster" /* ExpansionBooster */,
    setIdentifiers: ["cru"],
    talents: []
  },
  {
    cards: 40,
    classes: [Class.Warrior],
    deckLinks: [
      { url: "https://fabrary.net/decks/01G7FD2B3YQAMR8NJ4B3M58H96" }
    ],
    heroes: [Hero.Boltyn],
    languages: ["English" /* English */],
    release: Release.BoltynBlitzDeck,
    relatedReleases: [],
    releaseDate: "2021-05-07 12:00",
    releaseType: "Blitz Deck" /* BlitzDeck */,
    setIdentifiers: ["bol"],
    talents: [Talent.Light]
  },
  {
    cards: 40,
    classes: [Class.Runeblade],
    deckLinks: [
      { url: "https://fabrary.net/decks/01G7FDVRZP35DFWBRK64AG5TKQ" }
    ],
    heroes: [Hero.Chane],
    languages: ["English" /* English */],
    release: Release.ChaneBlitzDeck,
    relatedReleases: [],
    releaseDate: "2021-05-07 12:00",
    releaseType: "Blitz Deck" /* BlitzDeck */,
    setIdentifiers: ["chn"],
    talents: [Talent.Shadow]
  },
  {
    cards: 40,
    classes: [Class.Brute],
    languages: ["English" /* English */],
    deckLinks: [
      { url: "https://fabrary.net/decks/01G7B1T1D1M2DAM61K876VJBDK" }
    ],
    heroes: [Hero.Levia],
    release: Release.LeviaBlitzDeck,
    relatedReleases: [],
    releaseDate: "2021-05-07 12:00",
    releaseType: "Blitz Deck" /* BlitzDeck */,
    setIdentifiers: ["lev"],
    talents: [Talent.Shadow]
  },
  {
    cards: 40,
    classes: [Class.Illusionist],
    deckLinks: [
      { url: "https://fabrary.net/decks/01G7FCP2N7N0MNHWAH6JTP0KFN" }
    ],
    heroes: [Hero.Prism],
    languages: ["English" /* English */],
    relatedReleases: [],
    releaseDate: "2021-05-07 12:00",
    release: Release.PrismBlitzDeck,
    releaseType: "Blitz Deck" /* BlitzDeck */,
    setIdentifiers: ["psm"],
    talents: [Talent.Light]
  },
  {
    cards: 307,
    classes: [Class.Brute, Class.Illusionist, Class.Runeblade, Class.Warrior],
    deckLinks: [],
    draft: {
      heroIdentifiers: ["boltyn", "chane", "levia", "prism"],
      picksPerPack: 15
    },
    heroes: [Hero.Boltyn, Hero.Chane, Hero.Levia, Hero.Prism],
    languages: ["English" /* English */],
    release: Release.Monarch,
    relatedReleases: [
      Release.BoltynBlitzDeck,
      Release.ChaneBlitzDeck,
      Release.LeviaBlitzDeck,
      Release.PrismBlitzDeck
    ],
    releaseDate: "2021-05-07 12:00",
    releaseType: "Standalone Booster" /* StandaloneBooster */,
    setIdentifiers: ["mon"],
    talents: [Talent.Light, Talent.Shadow]
  },
  {
    cards: 40,
    classes: [Class.Runeblade],
    deckLinks: [
      { url: "https://fabrary.net/decks/01G7K4D304QQCZZSBT7ABCX4XC" }
    ],
    heroes: [Hero.Briar],
    languages: ["English" /* English */],
    release: Release.BriarBlitzDeck,
    relatedReleases: [],
    releaseDate: "2021-11-24 12:00",
    releaseType: "Blitz Deck" /* BlitzDeck */,
    talents: [Talent.Elemental, Talent.Earth, Talent.Lightning],
    setIdentifiers: ["bri"]
  },
  {
    cards: 40,
    classes: [Class.Ranger],
    deckLinks: [
      { url: "https://fabrary.net/decks/01G7K3WGPVKVDXG2J013GXSXNP" }
    ],
    heroes: [Hero.Lexi],
    languages: ["English" /* English */],
    release: Release.LexiBlitzDeck,
    relatedReleases: [],
    releaseDate: "2021-11-24 12:00",
    releaseType: "Blitz Deck" /* BlitzDeck */,
    setIdentifiers: ["lxi"],
    talents: [Talent.Elemental, Talent.Ice, Talent.Lightning]
  },
  {
    cards: 40,
    classes: [Class.Guardian],
    deckLinks: [
      { url: "https://fabrary.net/decks/01G7K464J7VS0K7HKW5E395TBK" }
    ],
    heroes: [Hero.Oldhim],
    languages: ["English" /* English */],
    release: Release.OldhimBlitzDeck,
    relatedReleases: [],
    releaseDate: "2021-11-24 12:00",
    releaseType: "Blitz Deck" /* BlitzDeck */,
    setIdentifiers: ["old"],
    talents: [Talent.Elemental, Talent.Earth, Talent.Ice]
  },
  {
    cards: 238,
    classes: [Class.Guardian, Class.Ranger, Class.Runeblade],
    deckLinks: [],
    draft: {
      heroIdentifiers: ["briar", "lexi", "oldhim"],
      picksPerPack: 15
    },
    heroes: [Hero.Briar, Hero.Lexi, Hero.Oldhim],
    languages: ["English" /* English */],
    release: Release.TalesOfAria,
    relatedReleases: [
      Release.BriarBlitzDeck,
      Release.LexiBlitzDeck,
      Release.OldhimBlitzDeck
    ],
    releaseDate: "2021-09-24 12:00",
    releaseType: "Standalone Booster" /* StandaloneBooster */,
    setIdentifiers: ["ele"],
    talents: [Talent.Elemental, Talent.Earth, Talent.Ice, Talent.Lightning]
  },
  {
    cards: 198,
    classes: [
      Class.Brute,
      Class.Guardian,
      Class.Illusionist,
      Class.Mechanologist,
      Class.Merchant,
      Class.Ninja,
      Class.Ranger,
      Class.Runeblade,
      Class.Warrior,
      Class.Wizard
    ],
    deckLinks: [],
    heroes: [Hero.Genis, Hero.Starvo, Hero.Valda],
    languages: ["English" /* English */],
    release: Release.Everfest,
    relatedReleases: [],
    releaseDate: "2022-02-04 12:00",
    releaseType: "Expansion Booster" /* ExpansionBooster */,
    setIdentifiers: ["evr"],
    talents: [Talent.Elemental, Talent.Earth, Talent.Ice, Talent.Lightning]
  },
  {
    cards: 423,
    classes: [
      Class.Brute,
      Class.Guardian,
      Class.Mechanologist,
      Class.Ninja,
      Class.Ranger,
      Class.Runeblade,
      Class.Warrior,
      Class.Wizard
    ],
    deckLinks: [],
    heroes: [
      Hero.Azalea,
      Hero.Benji,
      Hero.Bravo,
      Hero.Dash,
      Hero.DataDoll,
      Hero.Dorinthea,
      Hero.Ira,
      Hero.Kassai,
      Hero.Katsu,
      Hero.Kayo,
      Hero.Rhinar,
      Hero.Viserai,
      Hero.Kano
    ],
    languages: ["English" /* English */],
    release: Release.HistoryPack1,
    relatedReleases: [],
    releaseDate: "2022-05-06 12:00",
    releaseType: "History Pack" /* HistoryPack */,
    setIdentifiers: ["1hp", "hp1"],
    talents: []
  },
  {
    cards: 80,
    classes: [Class.Brute, Class.Warrior],
    deckLinks: [
      {
        name: "Dorinthea",
        url: "https://fabrary.net/decks/01GA4WTNSD12YTQB4FY8EYYNG5"
      },
      {
        name: "Rhinar",
        url: "https://fabrary.net/decks/01GA4WZKX9VEWKKPX7Y7H3MG1F"
      }
    ],
    heroes: [Hero.Dorinthea, Hero.Rhinar],
    languages: ["English" /* English */],
    release: Release.ClassicBattlesRhinarDorinthea,
    relatedReleases: [],
    releaseDate: "2022-05-27 12:00",
    releaseType: "Classic Battles" /* ClassicBattles */,
    setIdentifiers: ["dvr", "rvd"],
    talents: []
  },
  {
    cards: 40,
    classes: [Class.Illusionist],
    deckLinks: [
      { url: "https://fabrary.net/decks/01G76H1R1ERRBRKS7RVCQAB8RX" }
    ],
    heroes: [Hero.Dromai],
    languages: ["English" /* English */],
    relatedReleases: [],
    releaseDate: "2022-06-24 12:00",
    release: Release.DromaiBlitzDeck,
    releaseType: "Blitz Deck" /* BlitzDeck */,
    setIdentifiers: ["dro"],
    talents: [Talent.Draconic]
  },
  {
    cards: 40,
    classes: [Class.Ninja],
    deckLinks: [
      { url: "https://fabrary.net/decks/01G76H7RG7GN5ZA10F3BJBH740" }
    ],
    heroes: [Hero.Fai],
    languages: ["English" /* English */],
    release: Release.FaiBlitzDeck,
    relatedReleases: [],
    releaseDate: "2022-06-24 12:00",
    releaseType: "Blitz Deck" /* BlitzDeck */,
    setIdentifiers: ["fai"],
    talents: [Talent.Draconic]
  },
  {
    cards: 226,
    classes: [Class.Illusionist, Class.Ninja, Class.Wizard],
    deckLinks: [],
    draft: {
      heroIdentifiers: ["dromai", "fai", "iyslander"],
      picksPerPack: 14
    },
    heroes: [Hero.Dromai, Hero.Fai, Hero.Iyslander],
    languages: ["English" /* English */],
    release: Release.Uprising,
    relatedReleases: [Release.DromaiBlitzDeck, Release.FaiBlitzDeck],
    releaseDate: "2022-06-24 12:00",
    releaseType: "Standalone Booster" /* StandaloneBooster */,
    setIdentifiers: ["upr"],
    talents: [Talent.Draconic, Talent.Elemental, Talent.Ice]
  },
  {
    cards: 247,
    classes: [
      Class.Assassin,
      Class.Brute,
      Class.Guardian,
      Class.Illusionist,
      Class.Mechanologist,
      Class.Ninja,
      Class.Ranger,
      Class.Runeblade,
      Class.Warrior,
      Class.Wizard
    ],
    deckLinks: [],
    heroes: [Hero.Arakni, Hero.Emperor, Hero.Yoji],
    languages: ["English" /* English */],
    release: Release.Dynasty,
    relatedReleases: [],
    releaseDate: "2022-11-11 12:00",
    releaseType: "Expansion Booster" /* ExpansionBooster */,
    setIdentifiers: ["dyn"],
    talents: [Talent.Royal]
  },
  // {
  //   cards: 536,
  //   classes: [
  //     Class.Brute,
  //     Class.Guardian,
  //     Class.Illusionist,
  //     Class.Mechanologist,
  //     Class.Ninja,
  //     Class.Ranger,
  //     Class.Runeblade,
  //     Class.Warrior,
  //     Class.Wizard,
  //   ],
  //   deckLinks: [],
  //   heroes: [
  //     Hero.Boltyn,
  //     Hero.Briar,
  //     Hero.Chane,
  //     Hero.Levia,
  //     Hero.Lexi,
  //     Hero.Oldhim,
  //     Hero.Prism,
  //   ],
  //   languages: [
  //     Language.French,
  //     Language.German,
  //     Language.Italian,
  //     Language.Spanish,
  //   ],
  //   release: "History Pack 2: Black Label" as Release,
  //   relatedReleases: [],
  //   releaseDate: "2023-02-24 12:00",
  //   releaseType: ReleaseType.HistoryPack,
  //   setIdentifiers: ["hp2"],
  //   talents: [
  //     Talent.Elemental,
  //     Talent.Earth,
  //     Talent.Ice,
  //     Talent.Lightning,
  //     Talent.Light,
  //     Talent.Shadow,
  //   ],
  // },
  {
    cards: 40,
    classes: [Class.Assassin],
    deckLinks: [
      { url: "https://fabrary.net/decks/01GW25PNWBW2FPDHRNTPSET0GT" }
    ],
    heroes: [Hero.Arakni],
    languages: [
      "English" /* English */,
      "Fran\xE7ais" /* French */,
      "Deutsch" /* German */,
      "Italiano" /* Italian */,
      "Espa\xF1ol" /* Spanish */
    ],
    release: Release.ArakniBlitzDeck,
    relatedReleases: [],
    releaseDate: "2023-03-24 12:00",
    releaseType: "Blitz Deck" /* BlitzDeck */,
    setIdentifiers: ["ara"],
    talents: []
  },
  {
    cards: 40,
    classes: [Class.Ranger],
    deckLinks: [
      { url: "https://fabrary.net/decks/01GW26RNH1Q1J0S1VFXPN609S2" }
    ],
    heroes: [Hero.Azalea],
    languages: [
      "English" /* English */,
      "Fran\xE7ais" /* French */,
      "Deutsch" /* German */,
      "Italiano" /* Italian */,
      "Espa\xF1ol" /* Spanish */
    ],
    release: Release.AzaleaBlitzDeck,
    relatedReleases: [],
    releaseDate: "2023-03-24 12:00",
    releaseType: "Blitz Deck" /* BlitzDeck */,
    setIdentifiers: ["azl"],
    talents: []
  },
  {
    cards: 40,
    classes: [Class.Ninja],
    deckLinks: [
      { url: "https://fabrary.net/decks/01GW28622XSVAK7H9RXZVAVKM0" }
    ],
    heroes: [Hero.Benji],
    languages: [
      "English" /* English */,
      "Fran\xE7ais" /* French */,
      "Deutsch" /* German */,
      "Italiano" /* Italian */,
      "Espa\xF1ol" /* Spanish */
    ],
    release: Release.BenjiBlitzDeck,
    relatedReleases: [],
    releaseDate: "2023-03-24 12:00",
    releaseType: "Blitz Deck" /* BlitzDeck */,
    setIdentifiers: ["ben"],
    talents: []
  },
  {
    cards: 40,
    classes: [Class.Ninja],
    deckLinks: [
      { url: "https://fabrary.net/decks/01GW28TF5FXFKJ5CGRNXB84CG6" }
    ],
    heroes: [Hero.Katsu],
    languages: [
      "English" /* English */,
      "Fran\xE7ais" /* French */,
      "Deutsch" /* German */,
      "Italiano" /* Italian */,
      "Espa\xF1ol" /* Spanish */
    ],
    release: Release.KatsuBlitzDeck,
    relatedReleases: [],
    releaseDate: "2023-03-24 12:00",
    releaseType: "Blitz Deck" /* BlitzDeck */,
    setIdentifiers: ["kat"],
    talents: []
  },
  {
    cards: 40,
    classes: [Class.Ranger],
    deckLinks: [
      { url: "https://fabrary.net/decks/01GW29DPNZ3XVBJQHHFQHHHDGN" }
    ],
    heroes: [Hero.Riptide],
    languages: [
      "English" /* English */,
      "Fran\xE7ais" /* French */,
      "Deutsch" /* German */,
      "Italiano" /* Italian */,
      "Espa\xF1ol" /* Spanish */
    ],
    release: Release.RiptideBlitzDeck,
    relatedReleases: [],
    releaseDate: "2023-03-24 12:00",
    releaseType: "Blitz Deck" /* BlitzDeck */,
    setIdentifiers: ["rip"],
    talents: []
  },
  {
    cards: 40,
    classes: [Class.Assassin],
    deckLinks: [
      { url: "https://fabrary.net/decks/01GW2945GHPH2YSX3FTS7HCBT1" }
    ],
    heroes: [Hero.Uzuri],
    languages: [
      "English" /* English */,
      "Fran\xE7ais" /* French */,
      "Deutsch" /* German */,
      "Italiano" /* Italian */,
      "Espa\xF1ol" /* Spanish */
    ],
    release: Release.UzuriBlitzDeck,
    relatedReleases: [],
    releaseDate: "2023-03-24 12:00",
    releaseType: "Blitz Deck" /* BlitzDeck */,
    setIdentifiers: ["uzu"],
    talents: []
  },
  {
    cards: 239,
    classes: [Class.Assassin, Class.Ranger, Class.Ninja],
    deckLinks: [],
    draft: {
      heroIdentifiers: [
        "arakni-solitary-confinement",
        "azalea",
        "benji-the-piercing-wind",
        "katsu",
        "riptide",
        "uzuri"
      ],
      picksPerPack: 14
    },
    heroes: [
      Hero.Arakni,
      Hero.Azalea,
      Hero.Benji,
      Hero.Katsu,
      Hero.Riptide,
      Hero.Uzuri
    ],
    languages: [
      "English" /* English */,
      "Fran\xE7ais" /* French */,
      "Deutsch" /* German */,
      "Italiano" /* Italian */,
      "Espa\xF1ol" /* Spanish */
    ],
    release: Release.Outsiders,
    relatedReleases: [
      Release.ArakniBlitzDeck,
      Release.AzaleaBlitzDeck,
      Release.BenjiBlitzDeck,
      Release.KatsuBlitzDeck,
      Release.RiptideBlitzDeck,
      Release.UzuriBlitzDeck
    ],
    releaseDate: "2023-03-24 12:00",
    releaseType: "Standalone Booster" /* StandaloneBooster */,
    setIdentifiers: ["out"],
    talents: []
  },
  {
    cards: 236,
    classes: [Class.Illusionist, Class.Runeblade, Class.Warrior, Class.Brute],
    deckLinks: [],
    heroes: [Hero.Prism, Hero.Vynnset, Hero.Boltyn, Hero.Levia],
    languages: [
      "English" /* English */,
      "Fran\xE7ais" /* French */,
      "Deutsch" /* German */,
      "Italiano" /* Italian */,
      "Espa\xF1ol" /* Spanish */
    ],
    release: Release.DuskTillDawn,
    relatedReleases: [],
    releaseDate: "2023-07-14 12:00",
    releaseType: "Expansion Booster" /* ExpansionBooster */,
    setIdentifiers: ["dtd"],
    talents: [Talent.Light, Talent.Shadow]
  },
  {
    cards: 107,
    classes: [Class.Mechanologist, Class.Bard, Class.Guardian, Class.Ninja],
    deckLinks: [
      {
        name: "Brevant",
        url: "https://fabrary.net/decks/01HAXKPFH7ZHPDEJZH967VNRW6"
      },
      {
        name: "Ira",
        url: "https://fabrary.net/decks/01HAXKZMTRN4FR7CFTQA11A70N"
      },
      {
        name: "Melody",
        url: "https://fabrary.net/decks/01HAXKSFPXGCEH99PDZJ32V4N9"
      },
      {
        name: "Teklovossen",
        url: "https://fabrary.net/decks/01HAXKZMTRN4FR7CFTQA11A70N"
      }
    ],
    heroes: [Hero.Teklovossen, Hero.Melody, Hero.Brevant, Hero.Ira],
    languages: ["English" /* English */],
    release: Release.RoundTheTable,
    relatedReleases: [],
    releaseDate: "2023-09-29 12:00",
    releaseType: "Round the Table" /* RoundTheTable */,
    talents: [],
    setIdentifiers: ["tcc"]
  },
  {
    cards: 251,
    classes: [Class.Mechanologist],
    deckLinks: [],
    draft: {
      heroIdentifiers: ["dash-database", "maxx-nitro", "teklovossen"],
      picksPerPack: 14
    },
    heroes: [Hero.Dash, Hero.Maxx, Hero.Teklovossen],
    languages: [
      "English" /* English */,
      "Fran\xE7ais" /* French */,
      "Deutsch" /* German */,
      "Italiano" /* Italian */,
      "Espa\xF1ol" /* Spanish */
    ],
    raritiesExcludedInLimited: [Rarity.Legendary, Rarity.Fabled],
    release: Release.BrightLights,
    relatedReleases: [],
    releaseDate: "2023-10-06 12:00",
    releaseType: "Standalone Booster" /* StandaloneBooster */,
    setIdentifiers: ["evo"],
    talents: []
  },
  {
    cards: 40,
    classes: [Class.Guardian],
    deckLinks: [
      { url: "https://fabrary.net/decks/01HN3J1X1NQ9TB9RTVWV42JG46" }
    ],
    heroes: [Hero.Betsy],
    languages: [
      "English" /* English */,
      "Fran\xE7ais" /* French */,
      "Deutsch" /* German */,
      "Italiano" /* Italian */,
      "Espa\xF1ol" /* Spanish */
    ],
    release: Release.BetsyBlitzDeck,
    relatedReleases: [],
    releaseDate: "2024-02-02 12:00",
    releaseType: "Blitz Deck" /* BlitzDeck */,
    setIdentifiers: ["bet"],
    talents: []
  },
  {
    cards: 40,
    classes: [Class.Warrior],
    deckLinks: [
      { url: "https://fabrary.net/decks/01HN3SHBEQBRECYK6ZN1CHRADR" }
    ],
    heroes: [Hero.Kassai],
    languages: [
      "English" /* English */,
      "Fran\xE7ais" /* French */,
      "Deutsch" /* German */,
      "Italiano" /* Italian */,
      "Espa\xF1ol" /* Spanish */
    ],
    release: Release.KassaiBlitzDeck,
    relatedReleases: [],
    releaseDate: "2024-02-02 12:00",
    releaseType: "Blitz Deck" /* BlitzDeck */,
    setIdentifiers: ["ksi"],
    talents: []
  },
  {
    cards: 40,
    classes: [Class.Brute],
    deckLinks: [
      { url: "https://fabrary.net/decks/01HN3HK89NA910SK3FQV84H46Z" }
    ],
    heroes: [Hero.Kayo],
    languages: [
      "English" /* English */,
      "Fran\xE7ais" /* French */,
      "Deutsch" /* German */,
      "Italiano" /* Italian */,
      "Espa\xF1ol" /* Spanish */
    ],
    release: Release.KayoBlitzDeck,
    relatedReleases: [],
    releaseDate: "2024-02-02 12:00",
    releaseType: "Blitz Deck" /* BlitzDeck */,
    setIdentifiers: ["kyo"],
    talents: []
  },
  {
    cards: 40,
    classes: [Class.Warrior],
    deckLinks: [
      { url: "https://fabrary.net/decks/01HN3GP5ZJ1NRC0XBYNE45XQE8" }
    ],
    heroes: [Hero.Olympia],
    languages: [
      "English" /* English */,
      "Fran\xE7ais" /* French */,
      "Deutsch" /* German */,
      "Italiano" /* Italian */,
      "Espa\xF1ol" /* Spanish */
    ],
    release: Release.OlympiaBlitzDeck,
    relatedReleases: [],
    releaseDate: "2024-02-02 12:00",
    releaseType: "Blitz Deck" /* BlitzDeck */,
    setIdentifiers: ["ola"],
    talents: []
  },
  {
    cards: 40,
    classes: [Class.Brute],
    deckLinks: [
      { url: "https://fabrary.net/decks/01HN3Q4GBDMEFSX98HEN9ET308" }
    ],
    heroes: [Hero.Rhinar],
    languages: [
      "English" /* English */,
      "Fran\xE7ais" /* French */,
      "Deutsch" /* German */,
      "Italiano" /* Italian */,
      "Espa\xF1ol" /* Spanish */
    ],
    release: Release.RhinarBlitzDeck,
    relatedReleases: [],
    releaseDate: "2024-02-02 12:00",
    releaseType: "Blitz Deck" /* BlitzDeck */,
    setIdentifiers: ["rhi"],
    talents: []
  },
  {
    cards: 40,
    classes: [Class.Guardian],
    deckLinks: [
      { url: "https://fabrary.net/decks/01HN3K8ZV7KBRRJKPHW4B13J1K" }
    ],
    heroes: [Hero.Victor],
    languages: [
      "English" /* English */,
      "Fran\xE7ais" /* French */,
      "Deutsch" /* German */,
      "Italiano" /* Italian */,
      "Espa\xF1ol" /* Spanish */
    ],
    release: Release.VictorBlitzDeck,
    relatedReleases: [],
    releaseDate: "2024-02-02 12:00",
    releaseType: "Blitz Deck" /* BlitzDeck */,
    setIdentifiers: ["vic"],
    talents: []
  },
  {
    cards: 255,
    classes: [Class.Brute, Class.Guardian, Class.Warrior],
    deckLinks: [],
    draft: {
      heroIdentifiers: [
        "kassai",
        "betsy",
        "kayo",
        "olympia",
        "rhinar",
        "victor-goldmane"
      ],
      picksPerPack: 14
    },
    heroes: [
      Hero.Kassai,
      Hero.Betsy,
      Hero.Kayo,
      Hero.Olympia,
      Hero.Rhinar,
      Hero.Victor
    ],
    languages: [
      "English" /* English */,
      "Fran\xE7ais" /* French */,
      "Deutsch" /* German */,
      "Italiano" /* Italian */,
      "Espa\xF1ol" /* Spanish */
    ],
    raritiesExcludedInLimited: [Rarity.Legendary, Rarity.Fabled],
    release: Release.HeavyHitters,
    relatedReleases: [
      Release.BetsyBlitzDeck,
      Release.KassaiBlitzDeck,
      Release.KayoBlitzDeck,
      Release.OlympiaBlitzDeck,
      Release.RhinarBlitzDeck,
      Release.VictorBlitzDeck
    ],
    releaseDate: "2024-02-02 12:00",
    releaseType: "Standalone Booster" /* StandaloneBooster */,
    setIdentifiers: ["hvy"],
    talents: []
  },
  {
    cards: 66,
    classes: [Class.Brute],
    deckLinks: [
      { url: "https://fabrary.net/decks/01HWNCK2BYPVKK6701052YYXMZ" }
    ],
    heroes: [Hero.Kayo],
    languages: [
      "English" /* English */,
      "Fran\xE7ais" /* French */,
      "Italiano" /* Italian */,
      "\u65E5\u672C\u8A9E" /* Japanese */
    ],
    release: Release.ArmoryDeckKayo,
    relatedReleases: [],
    releaseDate: "2024-05-03 12:00",
    releaseType: "Armory Deck" /* ArmoryDeck */,
    setIdentifiers: ["ako"],
    talents: []
  },
  {
    cards: 40,
    classes: [Class.Illusionist],
    deckLinks: [
      { url: "https://fabrary.net/decks/01HX5Z23X2PRQCJ100FP4CAZGK" }
    ],
    heroes: [Hero.Enigma],
    languages: [
      "English" /* English */,
      "Fran\xE7ais" /* French */,
      "Deutsch" /* German */,
      "Italiano" /* Italian */,
      "\u65E5\u672C\u8A9E" /* Japanese */,
      "Espa\xF1ol" /* Spanish */
    ],
    release: Release.EnigmaBlitzDeck,
    relatedReleases: [],
    releaseDate: "2024-05-31 12:00",
    releaseType: "Blitz Deck" /* BlitzDeck */,
    setIdentifiers: ["eng"],
    talents: [Talent.Mystic]
  },
  {
    cards: 40,
    classes: [Class.Assassin],
    deckLinks: [
      { url: "https://fabrary.net/decks/01HX5Z4BD3TRMS3CP1ME4ZP3T6" }
    ],
    heroes: [Hero.Nuu],
    languages: [
      "English" /* English */,
      "Fran\xE7ais" /* French */,
      "Deutsch" /* German */,
      "Italiano" /* Italian */,
      "\u65E5\u672C\u8A9E" /* Japanese */,
      "Espa\xF1ol" /* Spanish */
    ],
    release: Release.NuuBlitzDeck,
    relatedReleases: [],
    releaseDate: "2024-05-31 12:00",
    releaseType: "Blitz Deck" /* BlitzDeck */,
    setIdentifiers: ["nuu"],
    talents: [Talent.Mystic]
  },
  {
    cards: 40,
    classes: [Class.Ninja],
    deckLinks: [
      { url: "https://fabrary.net/decks/01HX5Z395FD35CM24QCJQKJER1" }
    ],
    heroes: [Hero.Zen],
    languages: [
      "English" /* English */,
      "Fran\xE7ais" /* French */,
      "Deutsch" /* German */,
      "Italiano" /* Italian */,
      "\u65E5\u672C\u8A9E" /* Japanese */,
      "Espa\xF1ol" /* Spanish */
    ],
    release: Release.ZenBlitzDeck,
    relatedReleases: [],
    releaseDate: "2024-05-31 12:00",
    releaseType: "Blitz Deck" /* BlitzDeck */,
    setIdentifiers: ["zen"],
    talents: [Talent.Mystic]
  },
  {
    cards: 239,
    classes: [Class.Assassin, Class.Illusionist, Class.Ninja],
    deckLinks: [],
    draft: {
      heroIdentifiers: ["enigma", "nuu", "zen"],
      picksPerPack: 14
    },
    heroes: [Hero.Enigma, Hero.Nuu, Hero.Zen],
    languages: ALL_LANGUAGES_OLD,
    raritiesExcludedInLimited: [Rarity.Legendary, Rarity.Fabled],
    release: Release.PartTheMistveil,
    relatedReleases: [
      Release.EnigmaBlitzDeck,
      Release.NuuBlitzDeck,
      Release.ZenBlitzDeck
    ],
    releaseDate: "2024-05-31 12:00",
    releaseType: "Standalone Booster" /* StandaloneBooster */,
    setIdentifiers: ["mst"],
    talents: [Talent.Mystic]
  },
  {
    cards: 66,
    classes: [Class.Warrior],
    deckLinks: [
      { url: "https://fabrary.net/decks/01J202NH0RG8S0V8WXH1FWB2AH" }
    ],
    heroes: [Hero.Boltyn],
    languages: [
      "English" /* English */,
      "Fran\xE7ais" /* French */,
      "Italiano" /* Italian */,
      "\u65E5\u672C\u8A9E" /* Japanese */
    ],
    release: Release.ArmoryDeckBoltyn,
    relatedReleases: [],
    releaseDate: "2024-07-12 12:00",
    releaseType: "Armory Deck" /* ArmoryDeck */,
    setIdentifiers: ["asb"],
    talents: [Talent.Light]
  },
  {
    cards: 40,
    classes: [Class.Runeblade],
    deckLinks: [
      { url: "https://fabrary.net/decks/01J45SYSM2ZERW6CWY283ZAT1S" }
    ],
    heroes: [Hero.Aurora],
    languages: [
      "English" /* English */,
      "Fran\xE7ais" /* French */,
      "Deutsch" /* German */,
      "Italiano" /* Italian */,
      "\u65E5\u672C\u8A9E" /* Japanese */,
      "Espa\xF1ol" /* Spanish */
    ],
    release: Release.FirstStrikeAurora,
    relatedReleases: [],
    releaseDate: "2024-08-01 12:00",
    releaseType: "1st Strike" /* FirstStrike */,
    setIdentifiers: ["aur"],
    talents: [Talent.Lightning]
  },
  {
    cards: 40,
    classes: [Class.Guardian],
    deckLinks: [
      { url: "https://fabrary.net/decks/01J45SWXNDV5XD8PTBACAV8NZ1" }
    ],
    heroes: [Hero.Terra],
    languages: [
      "English" /* English */,
      "Fran\xE7ais" /* French */,
      "Deutsch" /* German */,
      "Italiano" /* Italian */,
      "\u65E5\u672C\u8A9E" /* Japanese */,
      "Espa\xF1ol" /* Spanish */
    ],
    release: Release.FirstStrikeTerra,
    relatedReleases: [],
    releaseDate: "2024-08-01 12:00",
    releaseType: "1st Strike" /* FirstStrike */,
    setIdentifiers: ["ter"],
    talents: [Talent.Earth]
  },
  {
    cards: 66,
    classes: [Class.Ranger],
    deckLinks: [
      { url: "https://fabrary.net/decks/01J3GKKSTM773CW7BG3RRJ5FJH" }
    ],
    heroes: [Hero.Azalea],
    languages: [
      "English" /* English */,
      "Fran\xE7ais" /* French */,
      "Italiano" /* Italian */,
      "\u65E5\u672C\u8A9E" /* Japanese */
    ],
    release: Release.ArmoryDeckAzalea,
    relatedReleases: [],
    releaseDate: "2024-08-02 12:00",
    releaseType: "Armory Deck" /* ArmoryDeck */,
    setIdentifiers: ["aaz"],
    talents: []
  },
  {
    cards: 40,
    classes: [Class.Runeblade],
    deckLinks: [
      { url: "https://fabrary.net/decks/01J68KF1TFNN62TC0G7M993FTY" }
    ],
    heroes: [Hero.Aurora],
    languages: [
      "English" /* English */,
      "Fran\xE7ais" /* French */,
      "Deutsch" /* German */,
      "Italiano" /* Italian */,
      "\u65E5\u672C\u8A9E" /* Japanese */,
      "Espa\xF1ol" /* Spanish */
    ],
    release: Release.AuroraBlitzDeck,
    relatedReleases: [],
    releaseDate: "2024-09-20 12:00",
    releaseType: "Blitz Deck" /* BlitzDeck */,
    setIdentifiers: ["aua"],
    talents: [Talent.Lightning]
  },
  {
    cards: 40,
    classes: [Class.Runeblade],
    deckLinks: [
      { url: "https://fabrary.net/decks/01J68KFRMJFKRQYJJ7THG1GMTB" }
    ],
    heroes: [Hero.Florian],
    languages: [
      "English" /* English */,
      "Fran\xE7ais" /* French */,
      "Deutsch" /* German */,
      "Italiano" /* Italian */,
      "\u65E5\u672C\u8A9E" /* Japanese */,
      "Espa\xF1ol" /* Spanish */
    ],
    release: Release.FlorianBlitzDeck,
    relatedReleases: [],
    releaseDate: "2024-09-20 12:00",
    releaseType: "Blitz Deck" /* BlitzDeck */,
    setIdentifiers: ["flr"],
    talents: [Talent.Earth]
  },
  {
    cards: 40,
    classes: [Class.Wizard],
    deckLinks: [
      { url: "https://fabrary.net/decks/01J68KGA3JVKFHEXGKYW0441CY" }
    ],
    heroes: [Hero.Oscilio],
    languages: [
      "English" /* English */,
      "Fran\xE7ais" /* French */,
      "Deutsch" /* German */,
      "Italiano" /* Italian */,
      "\u65E5\u672C\u8A9E" /* Japanese */,
      "Espa\xF1ol" /* Spanish */
    ],
    release: Release.OscilioBlitzDeck,
    relatedReleases: [],
    releaseDate: "2024-09-20 12:00",
    releaseType: "Blitz Deck" /* BlitzDeck */,
    setIdentifiers: ["osc"],
    talents: [Talent.Lightning]
  },
  {
    cards: 40,
    classes: [Class.Wizard],
    deckLinks: [
      { url: "https://fabrary.net/decks/01J68KGT3XD3HMVVRFT174QNMF" }
    ],
    heroes: [Hero.Verdance],
    languages: [
      "English" /* English */,
      "Fran\xE7ais" /* French */,
      "Deutsch" /* German */,
      "Italiano" /* Italian */,
      "\u65E5\u672C\u8A9E" /* Japanese */,
      "Espa\xF1ol" /* Spanish */
    ],
    release: Release.VerdanceBlitzDeck,
    relatedReleases: [],
    releaseDate: "2024-09-20 12:00",
    releaseType: "Blitz Deck" /* BlitzDeck */,
    setIdentifiers: ["ver"],
    talents: [Talent.Earth]
  },
  {
    cards: 258,
    classes: [Class.Runeblade, Class.Wizard],
    deckLinks: [],
    draft: {
      heroIdentifiers: ["aurora", "florian", "oscilio", "verdance"],
      picksPerPack: 14
    },
    heroes: [Hero.Aurora, Hero.Florian, Hero.Oscilio, Hero.Verdance],
    languages: ALL_LANGUAGES_OLD,
    raritiesExcludedInLimited: [Rarity.Legendary, Rarity.Fabled],
    release: Release.Rosetta,
    relatedReleases: [
      Release.AuroraBlitzDeck,
      Release.FlorianBlitzDeck,
      Release.OscilioBlitzDeck,
      Release.VerdanceBlitzDeck
    ],
    releaseDate: "2024-09-20 12:00",
    releaseType: "Standalone Booster" /* StandaloneBooster */,
    setIdentifiers: ["ros"],
    talents: [Talent.Elemental, Talent.Earth, Talent.Lightning]
  },
  {
    cards: 66,
    classes: [Class.Mechanologist],
    deckLinks: [
      { url: "https://fabrary.net/decks/01J9822H5PANJAFQVMC4TPK4Z1" }
    ],
    heroes: [Hero.Dash],
    languages: [
      "English" /* English */,
      "Fran\xE7ais" /* French */,
      "Italiano" /* Italian */,
      "\u65E5\u672C\u8A9E" /* Japanese */
    ],
    release: Release.ArmoryDeckDash,
    relatedReleases: [],
    releaseDate: "2024-10-18 12:00",
    releaseType: "Armory Deck" /* ArmoryDeck */,
    setIdentifiers: ["aio"],
    talents: []
  },
  {
    cards: 30,
    classes: [Class.Guardian],
    deckLinks: [
      { url: "https://fabrary.net/decks/01JCPPENK52DTRBJZMWQF8S0X2" }
    ],
    heroes: [Hero.Jarl],
    languages: [
      "English" /* English */,
      "Fran\xE7ais" /* French */,
      "Italiano" /* Italian */,
      "\u65E5\u672C\u8A9E" /* Japanese */
    ],
    release: Release.ArmoryDeckOriginsJarl,
    relatedReleases: [],
    releaseDate: "2024-11-29 12:00",
    releaseType: "Armory Deck" /* ArmoryDeck */,
    setIdentifiers: ["ajv"],
    talents: [Talent.Elemental, Talent.Earth, Talent.Ice]
  },
  {
    cards: 40,
    classes: [Class.Assassin],
    deckLinks: [],
    heroes: [Hero.Arakni],
    languages: ALL_LANGUAGES_OLD,
    release: Release.ArakniBlitzDeckARK,
    relatedReleases: [],
    releaseDate: "2025-01-31 12:00",
    releaseType: "Blitz Deck" /* BlitzDeck */,
    setIdentifiers: ["ark"],
    talents: []
  },
  {
    cards: 40,
    classes: [Class.Assassin],
    deckLinks: [],
    heroes: [Hero.Crackni],
    languages: ALL_LANGUAGES_OLD,
    release: Release.ArakniWebOfDeceitBlitzDeck,
    relatedReleases: [],
    releaseDate: "2025-01-31 12:00",
    releaseType: "Blitz Deck" /* BlitzDeck */,
    setIdentifiers: ["wod"],
    talents: []
  },
  {
    cards: 40,
    classes: [Class.Ninja],
    deckLinks: [],
    heroes: [Hero.Cindra],
    languages: ALL_LANGUAGES_OLD,
    release: Release.CindraBlitzDeck,
    relatedReleases: [],
    releaseDate: "2025-01-31 12:00",
    releaseType: "Blitz Deck" /* BlitzDeck */,
    setIdentifiers: ["cin"],
    talents: [Talent.Draconic, Talent.Royal]
  },
  {
    cards: 40,
    classes: [Class.Warrior],
    deckLinks: [],
    heroes: [Hero.Fang],
    languages: ALL_LANGUAGES_OLD,
    release: Release.FangBlitzDeck,
    relatedReleases: [],
    releaseDate: "2025-01-31 12:00",
    releaseType: "Blitz Deck" /* BlitzDeck */,
    setIdentifiers: ["fng"],
    talents: [Talent.Draconic, Talent.Royal]
  },
  {
    cards: 265,
    classes: [Class.Assassin, Class.Ninja, Class.Warrior],
    deckLinks: [],
    draft: {
      heroIdentifiers: ["arakni-web-of-deceit", "cindra", "fang"],
      picksPerPack: 14
    },
    heroes: [Hero.Arakni, Hero.Cindra, Hero.Fang],
    heroOverrides: {
      [Hero.Arakni]: Hero.Crackni
    },
    languages: ALL_LANGUAGES_OLD,
    raritiesExcludedInLimited: [Rarity.Legendary, Rarity.Fabled],
    release: Release.TheHunted,
    relatedReleases: [
      Release.ArakniBlitzDeckARK,
      Release.ArakniWebOfDeceitBlitzDeck,
      Release.CindraBlitzDeck,
      Release.FangBlitzDeck
    ],
    releaseDate: "2025-01-31 12:00",
    releaseType: "Standalone Booster" /* StandaloneBooster */,
    setIdentifiers: ["hnt"],
    talents: [Talent.Chaos, Talent.Draconic, Talent.Royal]
  },
  {
    cards: 28,
    classes: [Class.Runeblade],
    deckLinks: [
      { url: "https://fabrary.net/decks/01JN2DEG4X2V8DVMCWFBWQTTSC" }
    ],
    heroes: [Hero.Aurora],
    languages: ["English" /* English */, "Fran\xE7ais" /* French */, "\u65E5\u672C\u8A9E" /* Japanese */],
    release: Release.ArmoryDeckAurora,
    relatedReleases: [],
    releaseDate: "2025-03-14 12:00",
    releaseType: "Armory Deck" /* ArmoryDeck */,
    setIdentifiers: ["ast"],
    talents: [Talent.Elemental, Talent.Lightning]
  },
  {
    cards: 28,
    classes: [Class.Mechanologist],
    deckLinks: [
      { url: "https://fabrary.net/decks/01JRH0631MH5A9JPVGTP3TKJXN" }
    ],
    heroes: [Hero.Maxx],
    languages: ["English" /* English */, "Fran\xE7ais" /* French */, "\u65E5\u672C\u8A9E" /* Japanese */],
    release: Release.ArmoryDeckMaxx,
    relatedReleases: [],
    releaseDate: "2025-04-17 12:00",
    releaseType: "Armory Deck" /* ArmoryDeck */,
    setIdentifiers: ["amx"],
    talents: []
  },
  {
    cards: 30,
    classes: [Class.Necromancer, Class.Pirate],
    deckLinks: [
      { url: "https://fabrary.net/decks/01JVYZ0NCHP49HAP40C23P14E3" }
    ],
    heroes: [Hero.GravyBones],
    languages: ALL_LANGUAGES,
    release: Release.ArmoryDeckGravyBones,
    relatedReleases: [],
    releaseDate: "2025-05-30 12:00",
    releaseType: "Armory Deck" /* ArmoryDeck */,
    setIdentifiers: ["agb"],
    talents: []
  },
  {
    cards: 265,
    classes: [
      Class.Mechanologist,
      Class.Necromancer,
      Class.Pirate,
      Class.Ranger,
      Class.Thief
    ],
    deckLinks: [],
    draft: {
      heroIdentifiers: ["gravy-bones", "marlynn", "puffin", "scurv-stowaway"],
      picksPerPack: 14
    },
    heroes: [Hero.GravyBones, Hero.Marlynn, Hero.Puffin, Hero.Scurv],
    languages: ALL_LANGUAGES,
    raritiesExcludedInLimited: [Rarity.Fabled, Rarity.Legendary],
    release: Release.HighSeas,
    relatedReleases: [],
    releaseDate: "2025-06-06 12:00",
    releaseType: "Standalone Booster" /* StandaloneBooster */,
    setIdentifiers: ["sea"],
    talents: []
  },
  {
    cards: 30,
    classes: [Class.Ninja],
    deckLinks: [
      { url: "https://fabrary.net/decks/01JZ97KZ5TQV8E0FYMAM0XVNX7" }
    ],
    heroes: [Hero.Ira],
    languages: ALL_LANGUAGES,
    release: Release.ArmoryDeckIra,
    relatedReleases: [],
    releaseDate: "2025-07-11 12:00",
    releaseType: "Armory Deck" /* ArmoryDeck */,
    setIdentifiers: ["asr"],
    talents: []
  },
  {
    cards: 30,
    classes: [Class.Runeblade],
    deckLinks: [
      { url: "https://fabrary.net/decks/01K0ZQGQYJPH6B90VNT3GJ8C7Q" }
    ],
    heroes: [Hero.Viserai],
    languages: ALL_LANGUAGES,
    release: Release.ArmoryDeckLegendsViserai,
    relatedReleases: [],
    releaseDate: "2025-07-24 12:00",
    releaseType: "Armory Deck" /* ArmoryDeck */,
    setIdentifiers: ["avs"],
    talents: []
  },
  {
    cards: 30,
    classes: [Class.Illusionist],
    deckLinks: [
      { url: "https://fabrary.net/decks/01K0ZQ8FVR8DQB7YC9N7KAB7N8" }
    ],
    heroes: [Hero.Prism],
    languages: ALL_LANGUAGES,
    release: Release.ArmoryDeckLegendsPrism,
    relatedReleases: [],
    releaseDate: "2025-07-24 12:00",
    releaseType: "Armory Deck" /* ArmoryDeck */,
    setIdentifiers: ["apr"],
    talents: [Talent.Light]
  },
  {
    cards: 130,
    classes: [Class.Guardian],
    deckLinks: [],
    heroes: [Hero.Valda],
    languages: ALL_LANGUAGES,
    release: Release.MasteryPackGuardian,
    relatedReleases: [],
    releaseDate: "2025-08-08 12:00",
    releaseType: "Mastery Pack" /* MasteryPack */,
    setIdentifiers: ["mpg"],
    talents: []
  },
  {
    cards: 30,
    classes: [Class.Guardian],
    deckLinks: [
      { url: "https://fabrary.net/decks/01K1GBSGB17GT925NVJ0PM0G9Y" }
    ],
    heroes: [Hero.Bravo],
    languages: ALL_LANGUAGES,
    release: Release.BravoDemoDeck,
    relatedReleases: [],
    releaseDate: "2025-08-08 12:00",
    releaseType: "Welcome Deck" /* WelcomeDeck */,
    setIdentifiers: ["bdd"],
    talents: []
  },
  {
    cards: 107,
    classes: [Class.Guardian],
    deckLinks: [],
    heroes: [
      Hero.Bolfar,
      Hero.Bravo,
      Hero.Kox,
      Hero.Crix,
      Hero.Reya,
      Hero.Victor
    ],
    languages: ["English" /* English */],
    release: Release.SmashPalace,
    relatedReleases: [],
    releaseDate: "2025-09-29 12:00",
    releaseType: "Box Set" /* BoxSet */,
    talents: [],
    setIdentifiers: ["smp"]
  },
  {
    cards: 276,
    classes: [Class.Brute, Class.Guardian],
    deckLinks: [],
    draft: {
      heroIdentifiers: [
        "tuffnut",
        "pleiades",
        "kayo-strong-arm",
        "lyath-goldmane"
      ],
      picksPerPack: 14
    },
    heroes: [Hero.Kayo, Hero.Lyath, Hero.Pleiades, Hero.Tuffnut],
    heroOverrides: {
      [Hero.Kayo]: Hero.RKO
    },
    languages: ALL_LANGUAGES,
    raritiesExcludedInLimited: [Rarity.Fabled, Rarity.Legendary],
    release: Release.SuperSlam,
    relatedReleases: [],
    releaseDate: "2025-09-26 12:00",
    releaseType: "Standalone Booster" /* StandaloneBooster */,
    setIdentifiers: ["sup"],
    talents: [Talent.Revered, Talent.Reviled]
  },
  {
    cards: 30,
    classes: [Class.Guardian],
    deckLinks: [
      { url: "https://fabrary.net/decks/01K4XX1ERKXRYW8XHWE9BTAS4W" }
    ],
    heroes: [Hero.Pleiades],
    languages: ALL_LANGUAGES,
    release: Release.ArmoryDeckPleiades,
    relatedReleases: [],
    releaseDate: "2025-09-19 12:00",
    releaseType: "Armory Deck" /* ArmoryDeck */,
    setIdentifiers: ["aps"],
    talents: [Talent.Revered]
  },
  {
    cards: 30,
    classes: [Class.Brute],
    deckLinks: [
      { url: "https://fabrary.net/decks/01K74RSFG9RTVPVN534DZPJJNQ" }
    ],
    heroes: [Hero.Rhinar],
    languages: ALL_LANGUAGES,
    release: Release.ArmoryDeckRhinar,
    relatedReleases: [],
    releaseDate: "2025-11-14 12:00",
    releaseType: "Armory Deck" /* ArmoryDeck */,
    setIdentifiers: ["arr"],
    talents: []
  },
  {
    cards: 30,
    classes: [Class.Assassin],
    deckLinks: [
      { url: "https://fabrary.net/decks/01K74RXDPRPJT9YWGFVAPHECDF" }
    ],
    heroes: [Hero.Slippy],
    languages: ALL_LANGUAGES,
    release: Release.ArmoryDeckArakni,
    relatedReleases: [],
    releaseDate: "2025-12-15 12:00",
    releaseType: "Armory Deck" /* ArmoryDeck */,
    setIdentifiers: ["aac"],
    talents: [Talent.Chaos]
  },
  {
    cards: 348,
    classes: [
      Class.Assassin,
      Class.Brute,
      Class.Generic,
      Class.Guardian,
      Class.Illusionist,
      Class.Mechanologist,
      Class.Necromancer,
      Class.Ninja,
      Class.Pirate,
      Class.Ranger,
      Class.Runeblade,
      Class.Thief,
      Class.Warrior,
      Class.Wizard
    ],
    deckLinks: [],
    heroes: [],
    languages: ALL_LANGUAGES,
    release: Release.CompendiumOfRathe,
    relatedReleases: [],
    releaseDate: "2026-02-13 12:00",
    releaseType: "Expansion Booster" /* ExpansionBooster */,
    setIdentifiers: ["pen"],
    talents: [
      Talent.Chaos,
      Talent.Draconic,
      Talent.Earth,
      Talent.Elemental,
      Talent.Ice,
      Talent.Light,
      Talent.Lightning,
      Talent.Mystic,
      Talent.Revered,
      Talent.Reviled,
      Talent.Shadow
    ]
  },
  {
    cards: 40,
    classes: [Class.Assassin],
    deckLinks: [
      { url: "https://fabrary.net/decks/01KFWFQYASKA6C1FGXXB0E8EGE" }
    ],
    heroes: [Hero.Arakni],
    languages: ALL_LANGUAGES,
    release: Release.ArakniSilverAgeDeck,
    relatedReleases: [],
    releaseDate: "2026-02-13 12:00",
    releaseType: "Silver Age Deck" /* SilverAgeDeck */,
    setIdentifiers: ["sar"],
    talents: [Talent.Chaos]
  },
  {
    cards: 40,
    classes: [Class.Ranger],
    deckLinks: [
      { url: "https://fabrary.net/decks/01KFWFSGRBJS7BK87T8YF24FYC" }
    ],
    heroes: [Hero.Azalea],
    languages: ALL_LANGUAGES,
    release: Release.AzaleaSilverAgeDeck,
    relatedReleases: [],
    releaseDate: "2026-02-13 12:00",
    releaseType: "Silver Age Deck" /* SilverAgeDeck */,
    setIdentifiers: ["saz"],
    talents: []
  },
  {
    cards: 40,
    classes: [Class.Guardian],
    deckLinks: [
      { url: "https://fabrary.net/decks/01KFWFTPVMBMZ9TT5GVT5QRRRM" }
    ],
    heroes: [Hero.Bravo],
    languages: ALL_LANGUAGES,
    release: Release.BravoSilverAgeDeck,
    relatedReleases: [],
    releaseDate: "2026-02-13 12:00",
    releaseType: "Silver Age Deck" /* SilverAgeDeck */,
    setIdentifiers: ["sbr"],
    talents: []
  },
  {
    cards: 40,
    classes: [Class.Mechanologist],
    deckLinks: [
      { url: "https://fabrary.net/decks/01KFWFW6EAT7VX44MKN8GCVQSM" }
    ],
    heroes: [Hero.Dash],
    languages: ALL_LANGUAGES,
    release: Release.DashSilverAgeDeck,
    relatedReleases: [],
    releaseDate: "2026-02-13 12:00",
    releaseType: "Silver Age Deck" /* SilverAgeDeck */,
    setIdentifiers: ["sda"],
    talents: []
  },
  {
    cards: 40,
    classes: [Class.Warrior],
    deckLinks: [
      { url: "https://fabrary.net/decks/01KFWFXK3QM2XEVA08D5Y1R3CF" }
    ],
    heroes: [Hero.Dorinthea],
    languages: ALL_LANGUAGES,
    release: Release.DorintheaSilverAgeDeck,
    relatedReleases: [],
    releaseDate: "2026-02-13 12:00",
    releaseType: "Silver Age Deck" /* SilverAgeDeck */,
    setIdentifiers: ["sdo"],
    talents: []
  },
  {
    cards: 40,
    classes: [Class.Illusionist],
    deckLinks: [
      { url: "https://fabrary.net/decks/01KFWFYETE5DC1GK586XYXXFE6" }
    ],
    heroes: [Hero.Enigma],
    languages: ALL_LANGUAGES,
    release: Release.EnigmaSilverAgeDeck,
    relatedReleases: [],
    releaseDate: "2026-02-13 12:00",
    releaseType: "Silver Age Deck" /* SilverAgeDeck */,
    setIdentifiers: ["sen"],
    talents: [Talent.Mystic]
  },
  {
    cards: 40,
    classes: [Class.Ninja],
    deckLinks: [
      { url: "https://fabrary.net/decks/01KFWD89C2CQFA1TA331EJ1Q2K" }
    ],
    heroes: [Hero.Fai],
    languages: ALL_LANGUAGES,
    release: Release.FaiSilverAgeDeck,
    relatedReleases: [],
    releaseDate: "2026-02-13 12:00",
    releaseType: "Silver Age Deck" /* SilverAgeDeck */,
    setIdentifiers: ["sfa"],
    talents: [Talent.Draconic]
  },
  {
    cards: 40,
    classes: [Class.Wizard],
    deckLinks: [
      { url: "https://fabrary.net/decks/01KFWD9YWCRZ825T3AK8ZTG0G4" }
    ],
    heroes: [Hero.Iyslander],
    languages: ALL_LANGUAGES,
    release: Release.IyslanderSilverAgeDeck,
    relatedReleases: [],
    releaseDate: "2026-02-13 12:00",
    releaseType: "Silver Age Deck" /* SilverAgeDeck */,
    setIdentifiers: ["siy"],
    talents: [Talent.Elemental, Talent.Ice]
  },
  {
    cards: 40,
    classes: [Class.Brute],
    deckLinks: [
      { url: "https://fabrary.net/decks/01KFWFZ9Z5W17F0NHKQ44811Q0" }
    ],
    heroes: [Hero.Kayo],
    languages: ALL_LANGUAGES,
    release: Release.KayoSilverAgeDeck,
    relatedReleases: [],
    releaseDate: "2026-02-13 12:00",
    releaseType: "Silver Age Deck" /* SilverAgeDeck */,
    setIdentifiers: ["ska"],
    talents: []
  },
  {
    cards: 40,
    classes: [Class.Runeblade],
    deckLinks: [
      { url: "https://fabrary.net/decks/01KFWG05XQJT0NRVNGBZ4EJSSP" }
    ],
    heroes: [Hero.Viserai],
    languages: ALL_LANGUAGES,
    release: Release.ViseraiSilverAgeDeck,
    relatedReleases: [],
    releaseDate: "2026-02-13 12:00",
    releaseType: "Silver Age Deck" /* SilverAgeDeck */,
    setIdentifiers: ["svi"],
    talents: []
  },
  {
    cards: 27,
    classes: [Class.Warrior],
    deckLinks: [
      { url: "https://fabrary.net/decks/01KNHHE1MY39BC4PXYXMTJVT1M" }
    ],
    heroes: [Hero.Hala],
    languages: ALL_LANGUAGES,
    release: Release.ArmoryDeckOriginsHala,
    relatedReleases: [],
    releaseDate: "2026-04-17 12:00",
    releaseType: "Armory Deck" /* ArmoryDeck */,
    setIdentifiers: ["aha"],
    talents: []
  },
  {
    cards: 251,
    classes: [Class.Illusionist, Class.Runeblade, Class.Wizard],
    deckLinks: [],
    draft: {
      heroIdentifiers: [
        "aurora-emissary-of-lightning",
        "oscilio-scion-of-the-third-age",
        "zyggy"
      ],
      picksPerPack: 14
    },
    heroes: [Hero.Aurora2, Hero.Broscilio, Hero.Zyggy],
    heroOverrides: {
      [Hero.Aurora2]: Hero.Aurora,
      [Hero.Broscilio]: Hero.Oscilio
    },
    languages: ALL_LANGUAGES,
    raritiesExcludedInLimited: [Rarity.Legendary, Rarity.Fabled],
    release: Release.OmensOfTheThirdAge,
    relatedReleases: [],
    releaseDate: "2026-06-05 12:00",
    releaseType: "Standalone Booster" /* StandaloneBooster */,
    setIdentifiers: ["omn"],
    talents: [Talent.Lightning]
  },
  {
    cards: 40,
    classes: [Class.Illusionist],
    deckLinks: [
      { url: "https://fabrary.net/decks/01KREWW7RJS0GZ2PCCD4BM47QY" }
    ],
    heroes: [Hero.Zyggy],
    languages: ALL_LANGUAGES,
    release: Release.ArmoryDeckZyggy,
    relatedReleases: [],
    releaseDate: "2026-06-05 12:00",
    releaseType: "Armory Deck" /* ArmoryDeck */,
    setIdentifiers: ["azs"],
    talents: [Talent.Lightning]
  },
  {
    cards: 40,
    classes: [Class.Wizard],
    deckLinks: [
      { url: "https://fabrary.net/decks/01KP7ZQMB111K73XT88YV5YN7C" }
    ],
    heroes: [Hero.Blaze],
    languages: ALL_LANGUAGES,
    release: Release.BlazeSilverAgeDeck,
    relatedReleases: [],
    releaseDate: "2026-06-05 12:00",
    releaseType: "Silver Age Deck" /* SilverAgeDeck */,
    setIdentifiers: ["sbz"],
    talents: []
  },
  {
    cards: 40,
    classes: [Class.Warrior],
    deckLinks: [
      { url: "https://fabrary.net/decks/01KP7ZQ311EMGC2S97KC79AEX0" }
    ],
    heroes: [Hero.Boltyn],
    languages: ALL_LANGUAGES,
    release: Release.BoltynSilverAgeDeck,
    relatedReleases: [],
    releaseDate: "2026-06-05 12:00",
    releaseType: "Silver Age Deck" /* SilverAgeDeck */,
    setIdentifiers: ["sbl"],
    talents: [Talent.Light]
  },
  {
    cards: 40,
    classes: [Class.Runeblade],
    deckLinks: [
      { url: "https://fabrary.net/decks/01KP7ZP4HXQTBZ2G37PMJ9W20N" }
    ],
    heroes: [Hero.Briar],
    languages: ALL_LANGUAGES,
    release: Release.BriarSilverAgeDeck,
    relatedReleases: [],
    releaseDate: "2026-06-05 12:00",
    releaseType: "Silver Age Deck" /* SilverAgeDeck */,
    setIdentifiers: ["sba"],
    talents: [Talent.Elemental, Talent.Earth, Talent.Lightning]
  },
  {
    cards: 40,
    classes: [Class.Pirate, Class.Necromancer],
    deckLinks: [
      { url: "https://fabrary.net/decks/01KP7ZNJYM2QVMNRS4DQKFZHQ4" }
    ],
    heroes: [Hero.GravyBones],
    languages: ALL_LANGUAGES,
    release: Release.GravyBonesSilverAgeDeck,
    relatedReleases: [],
    releaseDate: "2026-06-05 12:00",
    releaseType: "Silver Age Deck" /* SilverAgeDeck */,
    setIdentifiers: ["sgb"],
    talents: []
  },
  {
    cards: 40,
    classes: [Class.Guardian],
    deckLinks: [
      { url: "https://fabrary.net/decks/01KP7ZMZGHMACW17CRP12E7R45" }
    ],
    heroes: [Hero.Lyath],
    languages: ALL_LANGUAGES,
    release: Release.LyathSilverAgeDeck,
    relatedReleases: [],
    releaseDate: "2026-06-05 12:00",
    releaseType: "Silver Age Deck" /* SilverAgeDeck */,
    setIdentifiers: ["sly"],
    talents: [Talent.Reviled]
  },
  {
    cards: 156,
    classes: [Class.Warrior],
    deckLinks: [],
    heroes: [Hero.Dorinthea],
    languages: ALL_LANGUAGES,
    release: Release.MasteryPackWarrior,
    relatedReleases: [],
    releaseDate: "2026-08-07 12:00",
    releaseType: "Mastery Pack" /* MasteryPack */,
    setIdentifiers: ["mpw"],
    talents: []
  },
  {
    cards: 30,
    classes: [Class.Warrior],
    deckLinks: [
      // { url: "https://fabrary.net/decks/01KP7ZJNFZZD8YNGP438FT8SFG" },
    ],
    heroes: [Hero.Olympia],
    languages: ALL_LANGUAGES,
    release: Release.ArmoryDeckOlympia,
    relatedReleases: [],
    releaseDate: "2026-08-07 12:00",
    releaseType: "Armory Deck" /* ArmoryDeck */,
    setIdentifiers: ["aol"],
    talents: []
  },
  {
    cards: 30,
    classes: [Class.Warrior],
    deckLinks: [
      { url: "https://fabrary.net/decks/01KP7ZHK1HMQCVMFH6W62FTVZ2" }
    ],
    heroes: [Hero.Dorinthea],
    languages: ALL_LANGUAGES,
    release: Release.DorintheaDemoDeck,
    relatedReleases: [],
    releaseDate: "2026-08-07 12:00",
    releaseType: "Welcome Deck" /* WelcomeDeck */,
    setIdentifiers: ["ddd"],
    talents: []
  },
  {
    cards: 250,
    classes: [Class.Runeblade],
    deckLinks: [
      // { url: "https://fabrary.net/decks/01KP7ZHK1HMQCVMFH6W62FTVZ2" },
    ],
    heroes: [Hero.Baalghor],
    languages: ALL_LANGUAGES,
    release: Release.IAR,
    relatedReleases: [],
    releaseDate: "2026-09-25 12:00",
    releaseType: "Expansion Booster" /* ExpansionBooster */,
    setIdentifiers: ["iar"],
    talents: [Talent.Shadow]
  }
];
const fullSetIdentifiers = {
  wtr: Release.WelcomeToRathe,
  arc: Release.ArcaneRising,
  cru: Release.CrucibleOfWar,
  mon: Release.Monarch,
  ele: Release.TalesOfAria,
  evr: Release.Everfest,
  "1hp": Release.HistoryPack1,
  hp1: Release.HistoryPack1,
  upr: Release.Uprising,
  dyn: Release.Dynasty,
  out: Release.Outsiders,
  dtd: Release.DuskTillDawn,
  evo: Release.BrightLights,
  hvy: Release.HeavyHitters,
  mst: Release.PartTheMistveil,
  ros: Release.Rosetta,
  hnt: Release.TheHunted,
  sea: Release.HighSeas,
  mpg: Release.MasteryPackGuardian,
  sup: Release.SuperSlam,
  pen: Release.CompendiumOfRathe,
  omn: Release.OmensOfTheThirdAge,
  mpw: Release.MasteryPackWarrior,
  iar: Release.IAR
};
const setIdentifierToSetMappings = {
  ...fullSetIdentifiers,
  // Armory decks
  aac: Release.ArmoryDeckArakni,
  aaz: Release.ArmoryDeckAzalea,
  asb: Release.ArmoryDeckBoltyn,
  aio: Release.ArmoryDeckDash,
  ako: Release.ArmoryDeckKayo,
  ajv: Release.ArmoryDeckOriginsJarl,
  ast: Release.ArmoryDeckAurora,
  amx: Release.ArmoryDeckMaxx,
  agb: Release.ArmoryDeckGravyBones,
  asr: Release.ArmoryDeckIra,
  aps: Release.ArmoryDeckPleiades,
  apr: Release.ArmoryDeckLegendsPrism,
  arr: Release.ArmoryDeckRhinar,
  avs: Release.ArmoryDeckLegendsViserai,
  aha: Release.ArmoryDeckOriginsHala,
  azs: Release.ArmoryDeckZyggy,
  aol: Release.ArmoryDeckOlympia,
  // Blitz decks
  ara: Release.ArakniBlitzDeck,
  ark: Release.ArakniBlitzDeckARK,
  aua: Release.AuroraBlitzDeck,
  azl: Release.AzaleaBlitzDeck,
  ben: Release.BenjiBlitzDeck,
  bet: Release.BetsyBlitzDeck,
  bol: Release.BoltynBlitzDeck,
  bri: Release.BriarBlitzDeck,
  chn: Release.ChaneBlitzDeck,
  cin: Release.CindraBlitzDeck,
  dro: Release.DromaiBlitzDeck,
  eng: Release.EnigmaBlitzDeck,
  fai: Release.FaiBlitzDeck,
  fng: Release.FangBlitzDeck,
  flr: Release.FlorianBlitzDeck,
  kat: Release.KatsuBlitzDeck,
  ksi: Release.KassaiBlitzDeck,
  kyo: Release.KayoBlitzDeck,
  lev: Release.LeviaBlitzDeck,
  lxi: Release.LexiBlitzDeck,
  nuu: Release.NuuBlitzDeck,
  old: Release.OldhimBlitzDeck,
  ola: Release.OlympiaBlitzDeck,
  osc: Release.OscilioBlitzDeck,
  psm: Release.PrismBlitzDeck,
  rip: Release.RiptideBlitzDeck,
  rhi: Release.RhinarBlitzDeck,
  uzu: Release.UzuriBlitzDeck,
  ver: Release.VerdanceBlitzDeck,
  vic: Release.VictorBlitzDeck,
  zen: Release.ZenBlitzDeck,
  wod: Release.ArakniWebOfDeceitBlitzDeck,
  // 1st strike decks
  aur: Release.FirstStrikeAurora,
  ter: Release.FirstStrikeTerra,
  // Silver Age decks
  sar: Release.ArakniSilverAgeDeck,
  saz: Release.AzaleaSilverAgeDeck,
  sba: Release.BriarSilverAgeDeck,
  sbl: Release.BoltynSilverAgeDeck,
  sbr: Release.BravoSilverAgeDeck,
  sbz: Release.BlazeSilverAgeDeck,
  sda: Release.DashSilverAgeDeck,
  sdo: Release.DorintheaSilverAgeDeck,
  sen: Release.EnigmaSilverAgeDeck,
  sfa: Release.FaiSilverAgeDeck,
  sgb: Release.GravyBonesSilverAgeDeck,
  siy: Release.IyslanderSilverAgeDeck,
  ska: Release.KayoSilverAgeDeck,
  sly: Release.LyathSilverAgeDeck,
  svi: Release.ViseraiSilverAgeDeck,
  // Starter decks
  bdd: Release.BravoDemoDeck,
  ddd: Release.DorintheaDemoDeck,
  dvr: Release.ClassicBattlesRhinarDorinthea,
  rvd: Release.ClassicBattlesRhinarDorinthea,
  ira: Release.IraWelcomeDeck,
  // Other decks
  tcc: Release.RoundTheTable,
  smp: Release.SmashPalace,
  // Hero decks
  bvo: Release.BravoHeroDeck,
  tea: Release.DorintheaHeroDeck,
  ksu: Release.KatsuHeroDeck,
  rnr: Release.RhinarHeroDeck,
  // GEM packs
  gem: Release.GEM,
  // Tournament packs
  tnp: Release.TournamentPack,
  // Promos
  anq: Release.Promos,
  con: Release.Promos,
  fab: Release.Promos,
  her: Release.Promos,
  jdg: Release.Promos,
  lgs: Release.Promos,
  lss: Release.Promos,
  oxo: Release.Promos,
  xxx: Release.Promos,
  win: Release.Promos
};
let tempSetToSetIdentifierMappings = {};
for (const [setIdentifier, set] of Object.entries(setIdentifierToSetMappings)) {
  const entry = tempSetToSetIdentifierMappings[set];
  if (entry) {
    entry.push(setIdentifier);
  } else {
    tempSetToSetIdentifierMappings[set] = [setIdentifier];
  }
}
const setToSetIdentifierMappings = tempSetToSetIdentifierMappings;
const silverAgeChapters = [
  {
    chapter: 1,
    name: "Silver Age Chapter 1",
    relatedReleases: [
      Release.BravoSilverAgeDeck,
      Release.DashSilverAgeDeck,
      Release.KayoSilverAgeDeck,
      Release.IyslanderSilverAgeDeck,
      Release.ViseraiSilverAgeDeck
    ],
    releaseDate: "2026-02-13 12:00"
  },
  {
    chapter: 2,
    name: "Silver Age Chapter 2",
    relatedReleases: [
      Release.ArakniSilverAgeDeck,
      Release.AzaleaSilverAgeDeck,
      Release.DorintheaSilverAgeDeck,
      Release.EnigmaSilverAgeDeck,
      Release.FaiSilverAgeDeck
    ],
    releaseDate: "2026-02-13 12:00"
  },
  {
    chapter: 3,
    name: "Silver Age Chapter 3",
    relatedReleases: [
      Release.BlazeSilverAgeDeck,
      Release.BoltynSilverAgeDeck,
      Release.BriarSilverAgeDeck,
      Release.GravyBonesSilverAgeDeck,
      Release.LyathSilverAgeDeck
    ],
    releaseDate: "2026-06-05 12:00"
  }
];
export {
  Language,
  ReleaseType,
  fullSetIdentifiers,
  getIsBooster,
  getIsDraftable,
  getIsPreconstructed,
  getIsReprint,
  releases,
  setIdentifierToSetMappings,
  setToSetIdentifierMappings,
  silverAgeChapters
};
